shell/devspace_start.sh

```diff
@@ -67,6 +67,10 @@ This is how you can work with it:
 if [[ -z $DEV_CONTAINER_LOGFILE ]] || [[ $DEVENV_DEV_TERMINAL == "true" ]]; then
   echo -e "$BANNER"
   bash
+elif [[ $E2E == "true" ]]; then
```

Comment:
personally i'd just do `[[ -z $E2E ]]` because a lot of people have different conventions with flags in bash. like in our test script we have some optional flags there and i always use them like:

```
SKIP_VALIDATE=0 make test
```